### PR TITLE
Improved the description of the Twig global variables

### PR DESCRIPTION
**Pull request by javiereguiluz**
I was reading this section and I realized that these variables weren't explained but just described with "useless" descriptions. I propose to explain them more deeply.

### CODE REVIEW

File: book/templating.rst [hunk 1]
```diff
@@ -1202,15 +1202,22 @@ instance which will give you access to some application specific variables
 automatically:
 
 ``app.security``
-    The security context.
+    The :class:`Symfony\Component\Security\Core\SecurityContext` object or
+    ``null`` if there is none.
 ``app.user``
-    The current user object.
+    The representation of the current user or ``null`` if there is none. The
+    value stored in this variable can be a :class:`Symfony\Component\Security\Core\User\UserInterface`
```
Comment (WouterJ) [inline book/templating.rst]: the backslashes have to be doubled

File: book/templating.rst [hunk 2]
```diff
@@ -1202,15 +1202,22 @@ instance which will give you access to some application specific variables
...
+    object, any other object which implements a ``__toString()`` method or even
+    a regular string.
 ``app.request``
-    The request object.
+    The :class:`Symfony\Component\HttpFoundation\Request` object that represents
```
Comment (WouterJ) [inline book/templating.rst]: same here

File: book/templating.rst [hunk 3]
```diff
@@ -1202,15 +1202,22 @@ instance which will give you access to some application specific variables
 automatically:
 
 ``app.security``
-    The security context.
+    The :class:`Symfony\Component\Security\Core\SecurityContext` object or
```
Comment (WouterJ) [inline book/templating.rst]: ...and here

**Comment by WouterJ** [comment]
Apart from the syntax issues, :+1:

**Comment by javiereguiluz** [comment]
Fixed the syntax issues. Thanks for the review!

**Comment by xabbuh** [comment]
Great improvements, Javier! 👍

**Pull request closed without merging**